web/src/scheduled_messages_overlay_ui.js

```diff
@@ -75,6 +75,8 @@ export function remove_scheduled_message_id(scheduled_msg_id) {
 }
 
 export function initialize() {
+    scheduled_messages.rerender_ui();
```

Comment:
We should just add a `top_left_corner.initialize` to call this, if there isn't one -- this code isn't related to the overlay, and I'd prefer not to have it live here.

Reply:
And to be clear, my thought is we'd have `ui_init.js` call:
```
scheduled_message_count = scheduled_messages.get_count();
top_left_corner.initialize({scheduled_message_count});
```

So that we don't add an additional import cycle.